README.md

```diff
@@ -66,13 +66,13 @@ If you wish to stick with the 2.X implementation, you can continue to pull from
 
 ## Specifications
 
-The 3.X version is the version of Pact-PHP, not the pact specification version that it supports.
-
-Pact-Php 3.X -> 9.X supports [Pact-Specification 2.X](https://github.com/pact-foundation/pact-specification/tree/version-2).
-Pact-Php 10.X supports:
-    * [Pact-Specification 2.X](https://github.com/pact-foundation/pact-specification/tree/version-2)
-    * [Pact-Specification 3.X](https://github.com/pact-foundation/pact-specification/tree/version-3).
-    * [Pact-Specification 4.X](https://github.com/pact-foundation/pact-specification/tree/version-4).
+| Specification | Supported  | Pact-PHP Version |
```

Comment:
Used table format from pact-js and pact-net.

I can't decide what to put into the `Status` column by myself. I just put the value that I think it make sense. Please discuss about it.

Since we use this table format, I think we can merge the `Versions` section above into this table (the next PR).

Reply:
Do you know why pact specification v1 and v1.1 are not mentioned? I remember I read the reason somewhere but I don't know where to find that.

Reply:
After having this discussion, I decided to put `v1` and `v1.1` into the table https://pact-foundation.slack.com/archives/C05HCLA3C93/p1712542161109069

Reply:
I assume for now we support only `v9`. How long do we support it? I don't know. I think we can leave this question for later.